keck_vnc_launcher.py

```diff
@@ -468,9 +468,22 @@ def launch_xterm(self, command, pw, title):
     def open_ssh_tunnel(self, server, username, password, ssh_pkey, remote_port, local_port=None):
 
         #get next local port if need be
+        #NOTE: Try up to 100 ports beyond
         if not local_port:
```

Comment:
This isn't vital, but while this form of the test is short and concise I've been taught to avoid it because it can have unintended consequences as many things evaluate to False (i.e. `local_port = 0`).  I suggest `if local_port is None:` instead.